Demo/src/main/java/com/paypal/android/ui/paypalweb/PayPalCheckoutView.kt

```diff
@@ -58,23 +61,40 @@ fun PayPalWebView(
             .padding(horizontal = contentPadding)
             .verticalScroll(scrollState)
     ) {
-        Step1_CreateOrder(uiState, viewModel)
+        Step1_WebOrNativeCheckoutHeader(viewModel)
+        Step2_CreateOrder(uiState, viewModel)
         if (uiState.isCreateOrderSuccessful) {
-            Step2_StartPayPalWebCheckout(uiState, viewModel)
+            Step3_StartPayPalCheckout(uiState, viewModel)
         }
         if (uiState.isPayPalWebCheckoutSuccessful) {
-            Step3_CompleteOrder(uiState, viewModel)
+            Step4_CompleteOrder(uiState, viewModel)
         }
         Spacer(modifier = Modifier.size(contentPadding))
     }
 }
 
 @Composable
-private fun Step1_CreateOrder(uiState: PayPalWebUiState, viewModel: PayPalWebViewModel) {
+fun Step1_WebOrNativeCheckoutHeader(viewModel: PayPalCheckoutViewModel) {
+    val uiState: PayPalUiState by viewModel.uiState.collectAsStateWithLifecycle()
```

Comment:
Any reason we create another StateFlow here? I wonder if this could cause a double render. A lot of the demo app is inspired by this [NowInAndroid app](https://github.com/android/nowinandroid/tree/main). You can see here the `collectAsStateWithLifecycle()` methods are only called [at the top level.](https://github.com/android/nowinandroid/blob/main/feature/foryou/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt#L113)